src/app/components/error-info/error-info.component.html

```diff
@@ -0,0 +1,15 @@
+<div id="notfound" class="color-back">
+    <div class="notfound">
+        <div class="notfound-404">
+            <h1><span></span>{{$ctrl.status}}</h1>
```

Comment:
why is there an empty span here??

Reply:
Because in this span I show the crying emoji when there is a error

Reply:
in that case, it should have a class in order to identify it properly